lms/djangoapps/discussion/rest_api/api.py

```diff
@@ -365,7 +364,6 @@ def _format_datetime(dt):
         "enable_in_context": course_config.enable_in_context,
         "group_at_subsection": course_config.plugin_configuration.get("group_at_subsection", False),
         'learners_tab_enabled': ENABLE_LEARNERS_TAB_IN_DISCUSSIONS_MFE.is_enabled(course_key),
-        "reason_codes_enabled": ENABLE_DISCUSSION_MODERATION_REASON_CODES.is_enabled(course_key),
```

Comment:
also need to remove this from the frontend?

Reply:
[Frontend PR](https://github.com/openedx/frontend-app-discussions/pull/615) has been merged.